docs/sanic/getting_started.rst

```diff
@@ -20,7 +20,9 @@ to true will stop that features installation.
 
 .. code-block:: bash
 
-    SANIC_NO_UVLOOP=true SANIC_NO_UJSON=true pip3 install sanic
+    $ export SANIC_NO_UVLOOP=true
+    $ export SANIC_NO_UJSON=true
+    $ pip3 install --no-binary :all: sanic
```

Comment:
I'd prefer a oneliner to avoid littering the environment. If multiple lines, maybe remove the dollar signs so that it can be copy&pasted?

Reply:
I believe previous one liner doesn't work for `fish`

Reply:
Updated commit as oneliner

Reply:
@yunstanford Support for this syntax was added in Fish 3.1, probably because it was a major incompatibility with other shells.